lib/main.dart

```diff
@@ -1,9 +1,16 @@
 import 'package:flutter/material.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
 
 import 'HomePage.dart';
 
-void main() {
-  runApp(MyApp());
+Future<void> main() async {
```

Comment:
Not sure if that makes a difference if `main` is `async`.

Reply:
Not really, it just enabled the usage of the `await` keyword: https://dart.dev/codelabs/async-await#working-with-futures-async-and-await